src/community/advanced-data-card.tsx

```diff
@@ -315,7 +315,10 @@ export const AdvancedDataCard = React.forwardRef<HTMLDivElement, AdvancedDataCar
                                 !hasFooter && !hasExtras ? styles.minHeight : ''
                             )}
                         >
-                            <Box paddingTop={8}>
+                            <Box
+                                paddingTop={8}
+                                className={!hasFooter && !hasExtras ? '' : styles.minHeightBox}
```

Comment:
why don't you add this class to the parent div instead?

Reply:
because this would affect the spacing of the box, making it smaller than it should be to complete the minimum height of the component, increasing the height of the footer

<img width="445" alt="Captura de tela 2024-04-29 165810" src="https://github.com/Telefonica/mistica-web/assets/90707957/27732960-b9c2-47e5-8703-8216d5deaf8d">